PostProcess/Common/interface/PostFXContext.hpp

```diff
@@ -115,7 +115,8 @@ class PostFXContext
 
     ResourceRegistry m_Resources{RESOURCE_IDENTIFIER_COUNT};
 
-    Uint32 m_CurrentFrameIndex = 0;
+    Uint32 m_CurrentFrameIndex           = 0;
+    bool   m_IsSupportedBaseVertexOffset = false;
```

Comment:
In the `ScreenSpaceReflection`, we have the `SupportedDeviceFeatures` struct that contains the features.
It makes sense to move this structure to the `PostFXOontext` and add this new member there.
Also, this the naming is not quite correct.
We want is not the Base Vertex Offset support, but `ShaderBaseVertexOffset`.
Also, adding the comment will be helfpufl.